src/main/resources/db/migration/V20221013123300__trigger_creation_1.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE locations
```

Comment:
We have possibility for checking if this column is created already.
Could you try please?
IF NOT EXISTS/IF EXISTS choose what you like